libvast/src/system/application.cpp

```diff
@@ -134,6 +134,9 @@ auto make_dump_command() {
   dump->add_subcommand("concepts", "print all registered concept definitions",
                        documentation::vast_dump_concepts,
                        opts("?vast.dump.concepts"));
+  dump->add_subcommand("models", "print all registered model definitions",
+                       documentation::vast_dump_concepts,
+                       opts("?vast.dump.concepts"));
```

Comment:
```suggestion
  dump->add_subcommand("models", "print all registered model definitions",
                       documentation::vast_dump_models,
                       opts("?vast.dump.models"));
```

And then the command docs are also missing.